docs/docusaurus/docs/guides/expectations/creating_custom_expectations/add_custom_parameters.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_label: 'Add custom parameters to Custom Expectations'
+title: 'Add custom parameters to Custom Expectations'
+id: add_custom_parameters
+description: Add custom parameters to Custom Expectations.
+---
+
+Use custom parameters in your Custom Expectations when you have data dependencies that can’t or shouldn’t be hardcoded. The Custom Expectation class determines how you implement custom parameters.
+
+Using custom parameters in your Custom Expectations can help you create powerful, business-specific validations and help you optimize your Great Expectations (GX) workflows.
```

Comment:
I would flip the order of these paragraphs; the second one feels like more of an opener.

```suggestion
Using custom parameters in your Custom Expectations can help you create powerful, business-specific validations and help you optimize your Great Expectations (GX) workflows. Custom parameters should be implemented when you have data dependencies that can’t or shouldn’t be hardcoded. The method of implementing custom parameters is specific to the Custom Expectation class being extended.
```

...I may have adjusted the wording a little bit, too.